### DISCUSSION TITLE
v2ray 在电信宽带下无法工作。

### DISCUSSION THREAD

**Discussion by tothemoon10080** [v2fly/v2ray-core#1596] [Q&A]
用手机（中国移动）移动数据时网站可正常访问 ，但是使用wifi（中国电信）时就不能访问境外网站。使用ip和在全局模式下访问百度则非常慢但至少可以访问。
有人有相同遭遇吗，寻求帮助。
vps 服务商是cloudcone 位置在美国洛杉矶

**Comment by kingwilliam** [comment]
连接问题有更多资料可提供吗?

但cloudcone不是一个好选择, 对岸vps我也试过不下十间, cloudcone真的是最差一间, 只可说价钱的确比较便宜. 如果有其他vps是你选择之内, 我会建议你转场.

**Reply by tothemoon10080** [reply]
我并不了解都有哪些优质的vps服务商，你有推荐吗。

**Reply by kingwilliam** [reply]
自家最後落户 搬瓦工 (vultr 我觉得也不错)

最後为什麽没用vultr? 就是欠了web ssh功能(其他人可能没了也没关系,但我一定要有).
因如只得ssh 如不幸设定错误, 如忘记了 ssh port 或 firewall block了 ssh, 还可在 web ssh补救.
但如vultr没了 webssh就只能重装os

参考:
[搬瓦工和 Vultr VPS 哪个更好 – 机房/线路/速度/性价比对比和选择](https://www.bandwagonhost.net/12142.html)

**Comment by shulun12** [comment]
问题解决没有，我也是，我手机移动的给电脑开热点可以上网，但是电脑连接电信的宽带就不行。同一个节点，就只是用的网络不一样

**Reply by tothemoon10080** [reply]
我没有找到解决办法，过一段时间问题就自己消失了。

我认为问题是由于国内运营商对访问国外网站的流量（被识别为代理流量）做限制导致的。

我尝试过
- 启用 TLS 加密
- 使用域名伪装（SNI 伪装）
- 更换端口和协议

但都没有起作用...

**Reply by kingwilliam** [reply]
纯个人经验 你的vps IP已给认定是科学上网 GFW不会全封锁这IP 只要有持续流量 就会封锁一段时间 由最初分钟计算 到之後以小时计算 解决方法就只有换IP或供应商

又是另一个人经验 日本IP比较容易封

**Reply by shulun12** [reply]
> 纯个人经验你的vps IP已给认定是科学上网 GFW不会全封锁这个IP持续有持续流量可以封锁小时一段时间由最初分钟计算到之后以计算解决方法就只有换IP或供应商
> 
> 又是另一个人的经验日本IP比较容易封

我用的事aws的。下午2点到4点断断续续的，晚上11点之后也用不了，没有很具体的时候，就是晚上总有时间是用不了的，用不了的时候我ping了一下ip，是超时的，丢包很严重，百分之75的样子，很奇怪，不知道怎么回事，我换了好几个ip了，日本新加坡的都试了，都没用，电信宽带不能用的时候我用手机的热点又可以了，但是有时候也是不行的，不知道为什么这么问题，最开始搭建的时候没这么多问题，最多就是用了2个多月的时候就要重新搭建一次，ip给封了。

**Comment by swuecho** [comment]
出现了类似的问题，

我的是联通的宽带不行了，但是电信的移动wifi还行。

**Comment by tursom** [comment]
cloudcone 用 IPv6 访问的体验还不错，稳定使用大半年了
